THIRD-PARTY-NOTICES.TXT

```diff
@@ -1072,4 +1072,30 @@ Copyright (c) Microsoft Corporation.
 Licensed under the MIT License.
 
 Available at
-https://github.com/microsoft/msquic/blob/main/LICENSE
\ No newline at end of file
+https://github.com/microsoft/msquic/blob/main/LICENSE
+
+License notice for m-ou-se/floatconv
```

Comment:
Where did you use code licensed for `m-ou-se/floatconv`?

Reply:
In https://github.com/dotnet/runtime/pull/69204/files#diff-68eb1454c605f9d2192816343fd055ac5482a7b001e1f010f39c9ab1777ad441R252 (and another place) where the corresponding same callouts exist.